ChromecastSampleApp/ChromecastSampleApp/Players/PlayerViewController.m

```diff
@@ -87,7 +87,9 @@ - (void)viewDidLoad {
   // Init the castManager in the ooyalaPlayer
   [self.ooyalaPlayer initCastManager:self.castManager];
   [self.ooyalaPlayer setEmbedCode:self.embedCode];
-  [self.ooyalaPlayer play];
+  if( self.mediaInfo.startPlaying ) {
```

Comment:
i imagine instead of if self.mediaInfo.startPlaying, we check if manager.getCastPlayer.embedCode == self.embedCode

if they equal the same, we do not need to auto-call play.  if they are different (or if getCastPlayer is null we can auto-call play.

method would be something like "[self isCastPlayerCurrentlyDisplayingDesiredContent]", and if that's true, do not autoplay